src/lib/drivers/rest/autopilot/parameters/mod.rs

```diff
@@ -10,7 +10,12 @@ static ASSETS_DIR: Dir = include_dir!(
 pub fn parameters() -> HashMap<String, Vec<String>> {
     let mut parameters: HashMap<String, Vec<String>> = HashMap::new();
     for dir in ASSETS_DIR.dirs() {
-        let directory = dir.path().file_name().unwrap().to_str().unwrap();
+        let directory = dir
+            .path()
+            .file_name()
+            .unwrap()
+            .to_str()
+            .expect("Files should exist in compile time");
```

Comment:
oops, it's not the to_str that will fail here. You can either use:
- `dir.path().file_name().and_then(|dir| dir.to_str()).expect("Files should exist in compile time");`
- `dir.path().file_name().expect("Files should exist in compile time").to_str().unwrap();`

Reply:
ops!

Reply:
Check now